identity-service/src/routes/recaptcha.js

```diff
@@ -0,0 +1,41 @@
+const config = require('../config')
+const { handleResponse, successResponse, errorResponseForbidden } = require('../apiHelpers')
+const models = require('../models')
+
+module.exports = function (app) {
+  app.get('/captchascore', handleResponse(async (req, res, next) => {
+    if (req.headers['x-captcha-score'] !== config.get('captchaScoreSecret')) {
+      return errorResponseForbidden('Not permissioned to view captcha score.')
+    }
+
+    // TODO: make it take an array of userIds?
+    const libs = req.app.get('audiusLibs')
+    let response = {}
+    let queryParams = req.query
+
+    let userId
+    if (queryParams && queryParams.id) {
+      userId = parseInt(queryParams.id)
+    }
+
+    if (!userId) return successResponse({ msg: `userId is bad: ${userId}` })
```

Comment:
maybe something more informative than `bad` would be good -- missing userId param or something